media/js/firefox/lockwise/lockwise.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+
+    if (client._isFirefox() === true && client._getFirefoxMajorVersion() >= '70') {
```

Comment:
Nit: can simplify the conditional a bit here:
```suggestion
    if (client.isFirefox && client._getFirefoxMajorVersion() >= '70') {
```